components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/util/ActionExecutorConfig.java

```diff
@@ -73,6 +80,35 @@ public boolean isExecutionForActionTypeEnabled(ActionType actionType) {
         }
     }
 
+    public int getHttpReadTimeout() {
+
+        return parseTimeoutConfig(HTTP_READ_TIMEOUT_PROPERTY, DEFAULT_HTTP_READ_TIMEOUT);
+    }
+
+    public int getHttpConnectionRequestTimeout() {
+
+        return parseTimeoutConfig(HTTP_CONNECTION_REQUEST_TIMEOUT_PROPERTY, DEFAULT_HTTP_CONNECTION_REQUEST_TIMEOUT);
+    }
+
+    public int getHttpConnectionTimeout() {
+
+        return parseTimeoutConfig(HTTP_CONNECTION_TIMEOUT_PROPERTY, DEFAULT_HTTP_CONNECTION_TIMEOUT);
+    }
```

Comment:
Let's add method comments for these public methods

Reply:
Updated in https://github.com/wso2/carbon-identity-framework/pull/5935/commits/e89ea4c2346f44139605a36f742988f5a0e4e84b